xml/System.Security.Cryptography.Pkcs/SubjectIdentifier.xml

```diff
@@ -52,10 +52,21 @@
         <Parameter Name="certificate" Type="System.Security.Cryptography.X509Certificates.X509Certificate2" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="certificate">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="certificate">The certificate to verify for a match.</param>
+        <summary>Verifies if the specified certificate was used for the decryption operation.</summary>
+        <returns><see langword="true" /> if the certificate; <see langword="false" /> otherwise.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+If the current subject identifier instance is of type <xref:System.Security.Cryptography.SubjectIdentifierType.Pkcs.IssuerAndSerialNumber>, this method compares the current instance's X09 issuer serial number and issuer name with the specified certificate's issuer serial number and issuer name.
+
```

Comment:
No paragraph break between sentences.

Reply:
Not sure what you mean. Can you elaborate?

Reply:
What I meant, @carlossanlop, is that the two sentences should be in a single paragraph.

Reply:
Done.